Maint/replace edb with pyedb (#4269)
ansys/pyaedt: doc/source/User_guide/intro.rst

    ...
    cir.release_desktop(save_project=True, close_desktop=True)
    # Desktop is released here.

> Ansys EDB proprietary layout format is accessible through pyaedt using the following

Reviewer: ```suggestion
The Ansys EDB proprietary layout format is accessible through PyAEDT using this
```

code:

.. code:: python

    # Launch the latest installed version of AEDB.
    import pyaedt
    edb = pyaedt.Edb("mylayout.aedb")

>     # User can launch Edb directly from PyEDB class.

Reviewer: ```suggestion
    # You can also launch EDB directly from PyEDB.
```
